testing/fixtures/fixutres.go

```diff
@@ -0,0 +1,74 @@
+package fixtures
+
+import (
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/fossas/fossa-cli/files"
+	git "gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+// ProjectFixture describes a git locator for a test fixture and env vars for when it is to be built
+type ProjectFixture struct {
+	Name   string
+	URL    string
+	Commit string
+	Env    map[string]string
+}
+
+// FixtureDirectory returns the directory under which tests fixtures should be placed
+func FixtureDirectory() string {
```

Comment:
I'd name this `Directory` so it doesn't stutter (import will be `fixtures.Directory()`).